tools/sdbusplus/structNode.py

```
@@ -0,0 +1,43 @@
+        print("  add StructNode segments="+str(segments)+" intf name="+intf.name, file=sys.stderr)
+
+        if len(segments) == 0:
+            self.interface = intf
+            return;
+
+        if segments[0] == self.nodeName and len(segments) >= 2:
+            segments = segments[1:]
+
+        seg = segments[0]
+
+        if not seg in self.members:
```

Flake8 (7.3.0)

[error] test for membership should be 'not in'

(E713)